glitch-in-the-matrix/src/lib.rs

```diff
@@ -62,18 +63,29 @@ pub struct MatrixClient {
 }
 impl MatrixClient {
     pub fn new_from_access_token(token: &str, url: &str, hdl: &Handle) -> impl Future<Item = Self, Error = MatrixError> {
-        let conn = match HttpsConnector::new(4, hdl) {
+        let conn = match HttpsConnector::new(4) {
```

Comment:
The handle isn't used anymore in client creation. It's just stored for later use in `Drop`.